MISRA.md

```diff
@@ -31,6 +31,31 @@ _Ref 8.9.1_
        order of execution, some variables have file scope definitions rather
        than function scope.
 
+#### Rule 8.13
+_Ref 8.13.1_
+
+- MISRA C-2012 Rule 8.13 Parameter passed is never used, should be declared as
```

Comment:
This should be re-worded to be more generic. Something like:

MISRA C-2012 Rule 8.13 encourages the use of const-qualified types whenever possible. However, the FreeRTOS API requires that all task entry functions conform to the same function signature (TaskFunction_t) which includes a single void * pointer argument. Due to this restriction, it is not possible to use a const qualifier for the entry point function to any FreeRTOS task.

Reply:
I think it was fixed with this commit: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/529/commits/81e90d8ac5bf905159c46b0675b957876336c182

Reply:
IMO: It's too specific to the particular instance. We're likely to run into this exact violation quite frequently.

Reply:
it is ok to be specific might be even desirable, we do support multiple References for specific cases